libethcore/Miner.h

```diff
@@ -101,6 +101,7 @@ struct CUSettings : public MinerSettings
     unsigned schedule = 4;
     unsigned gridSize = 8192;
     unsigned blockSize = 128;
+    int usleep = -1;
```

Comment:
C and C++ are annoying languages, and even though `int` is usually 32-bits large, on some platforms, it can be 16-bits, so it's better to use one of the types in [`<cstdint>`](https://en.cppreference.com/w/cpp/types/integer) instead.

The following should work if you include `#include <cstdint>` at the top of the file. Alternatively, you could replace the `CLI::Range(-1, 1000000))` to a smaller range that will fit in a `int16` like `CLI::Range(-1, 10000))`.

```suggestion
    int_fast32_t usleep = -1;
```